src/main/java/edu/unc/lib/boxc/migration/cdm/services/MigrationProjectFactory.java

```diff
@@ -21,10 +21,18 @@ public class MigrationProjectFactory {
     private MigrationProjectFactory() {
     }
 
-    public static MigrationProject createMigrationProject(Path path, String name,
-                                                          String collectionId, String user, String cdmEnvId)
+    public static MigrationProject createCdmMigrationProject(Path path, String name,
+                                                             String collectionId, String user,
+                                                             String cdmEnvId, String bxcEnvId)
             throws IOException {
-        return createMigrationProject(path, name, collectionId, user, cdmEnvId, null);
+        return createMigrationProject(path, name, collectionId, user, cdmEnvId, bxcEnvId, "cdm");
```

Comment:
I think we should make constants like PROJECT_SOURCE_CDM for "cdm" and "files", and use those everywhere to try to ensure consistency. I'm trying to think of the best place to put them. It might make sense in either MigrationProject or MigrationProjectProperties, or in a new constants class of some sort.